src/test/java/org/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -381,7 +382,7 @@ private void assertWrong(BibDatabaseContext context) {
                 createBibtexKeyPatternPreferences(),
                 new JournalAbbreviationRepository(new Abbreviation("IEEE Software", "IEEE SW")), true)
                 .checkBibtexDatabase();
-        assertFalse(messages.toString(), messages.isEmpty());
+        assertFalse(messages.isEmpty(), messages.toString());
```

Comment:
Its always better to compare values to empty collections or strings then to use `assertFalse(...isEmpty())` because you get better error message in case the test fails. Of course, you didn't write this code but please change it to `assertEquals("", messages)`.

Reply:
Damn...ok leave it like it was before, but this is something we also have to fix in the future...